tools/buildtools/ambertools/acpype_Amber2Gromacs.xml

```diff
@@ -14,13 +14,12 @@
         
     ]]></command>
     <inputs>
-        <param name="input1" type="data" format="prmtop" label="Amber Topology file" help="Topology file (prmtop) generated in Tleap"/>
-        <param name="input2" type="data" format="inpcrd" label="Amber Coordinate file" help="Coordinate file (inpcrd) generated in Tleap"/>
```

Comment:
we should only use official Galaxy filetypes here

Reply:
Not sure how to make those available. They should be available though, since the tool "MMGBSA/MMPBSA tool for estimating ligand binding affinities" uses both of these amber filetypes as inputs.

Reply:
They just specify `format="txt"` though.

Reply:
Ok, that's fine, I can change the input formats to "format=txt" if it's more appropriate. this should be ok since these are just text files and amber/acpype should process them irrespective of extension. Amber users generally just add .inpcrd and .prmtop to files so one can readily know what the contents are without opening them. at least that's been my experience.

Reply:
Yes, Galaxy does not expose file extensions directly to users, just the Galaxy 'datatypes'. If you think it would be more intuitive to have prmtop and inpcrd available as datatypes, feel free to add them to the upstream repo, there is some documentation here: https://docs.galaxyproject.org/en/latest/dev/data_types.html

Reply:
Should be ok by just changing the formats to txt. amber/acpype will process the files the files either way, irrespective of extension. I might have to add new data types later on for other functions, but for now this should be ok :).